docs/plugins/lastgenre.rst

```diff
@@ -213,5 +213,9 @@ fetch genres for albums or items matching a certain query.
 By default, ``beet lastgenre`` matches albums. To match individual tracks or
 singletons, use the ``-A`` switch: ``beet lastgenre -A [QUERY]``.
 
+To preview the changes that would be made without applying them, use the ``-p``
+or ``--pretend`` flag. This shows which genres would be set but does not write
+or store any changes.
+
```

Comment:
Please also add the pretend option to the list of possible options further down in the docs!